integrated_channels/integrated_channel/admin/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""
+Django admin integration for integrated_channel app.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.contrib import admin
+
+from integrated_channels.integrated_channel.models import (
+    EnterpriseCustomerPluginConfiguration, EnterpriseIntegratedChannel
+)
+
+
+@admin.register(EnterpriseIntegratedChannel)
+class EnterpriseIntegratedChannelAdmin(admin.ModelAdmin):
+    """
+    Django admin model for EnterpriseIntegratedChannel.
+    """
+    list_display = (
+        "name",
+        "data_type",
+    )
+
+    list_filter = ("data_type",)
+    search_fields = ("name", "data_type",)
+
+    class Meta(object):
+        model = EnterpriseIntegratedChannel
+
+
+@admin.register(EnterpriseCustomerPluginConfiguration)
+class EnterpriseCustomerPluginConfigurationAdmin(admin.ModelAdmin):
+    """
+    Django admin model for EnterpriseCustomerPluginConfiguration.
+    """
+
+    fields = (
+        "enterprise_customer_uuid",
+        "enterprise_integrated_channel",
+        "active",
+    )
+
+    list_display = (
+        "enterprise_customer_name",
+        "integrated_channel_name",
+        "data_type",
+        "active",
+    )
+
+    readonly_fields = (
+        "integrated_channel_name",
+        "data_type",
+        "enterprise_customer_name",
+    )
+
+    list_filter = ("enterprise_integrated_channel", "active",)
+    search_fields = ("enterprise_customer_name", "integrated_channel_name",)
+
+    class Meta(object):
+        model = EnterpriseCustomerPluginConfiguration
+
+    def integrated_channel_name(self, ent_customer_plugin_config):
```

Comment:
Decorate it with `@property`, and Python will call the function when you just use it - example:

```python
from uuid import uuid4

class Thing:
    @property
    def real_static_value(self):
        return uuid4().hex

>>> x = Thing()
>>> x.real_static_value
    '34eb598ca3434f5fb949397528eb0580'
```

But it only works for functions where the only argument is `self`.